cmd/bedMaxWig/bedMaxWig.go

```diff
@@ -29,8 +42,16 @@ func bedMaxWig(infile string, database string, chromsizeFile string, outfile str
 				if currentBed.FieldsInitialized < 7 {
 					currentBed.FieldsInitialized = 7
 				}
-				currentBed.Annotation = append(currentBed.Annotation, fmt.Sprintf("%f", bedRangeMax(chromSlice, records[k].ChromStart, records[k].ChromEnd)))
-				outlist = append(outlist, currentBed)
+				if norm == true {
+					maxWig := bedRangeMax(chromSlice, records[k].ChromStart, records[k].ChromEnd)
+					normMaxWig := maxWig / wigTotal
```

Comment:
I think this is the only statement that needs to be different when norm == true.  I think you could have:
1) calc the max value with bedRangeMax
2) have an "if norm == true" that overwrites the max value with the max value divided by wigTotal
3) add the max value to the bed (might have been normalized in the previous step)
4) add this bed to the outlist.

Reply:
Ooo I think I understand. I would have maxWig be calculated, then an if statement for if we do math on it, and then lines 48 and 49, which would allow me to remove lines 50-52. I will keep this in mind and will change in a future bedMaxWig.go PR.